### PR TITLE
pokemon card grid restyled

### LINKED ISSUE #41: Pokemon Card Grid Restyled
**Is your feature request related to a problem? Please describe.**
The pokemon card grid is not resizable.

**Describe the solution you'd like**
Restyling the SCSS

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Fixed the grid resizing issue

Closes #41

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Clean code

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -8,43 +8,56 @@ const PokemonCard = ({ name, id, types, picture, pokemon, setSelectedPokemon })
...
+  const cardImage = {
+    backgroundImage: `url(${picture})`,
+    backgroundSize: `50%`,
+    backgroundColor: `hsl(0, 0%, 11%)`,
+    backgroundRepeat: `no-repeat`,
+    backgroundPosition: `center`,
+    width: `100%`,
+    height: `75%`,
+    borderRadius: `10px`,
+  }
```
Comment (BlackCubes) [inline pokeapp/src/components/PokemonCard.js:20]: WOAH!!!!!!!!!!!!!!!!!!

File: pokeapp/src/utilities/ApiCall.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const interval = {
 };
 
 const customOptions = {
-  protocol: "http",
+  protocol: "https",
```
Comment (BlackCubes) [inline pokeapp/src/utilities/ApiCall.js:9]: Das rite! Security!

**Review by BlackCubes — APPROVED** [review]
Looks clean and ready to go!

**Pull request merged**